lib/inspec/resources/filesystem.rb

```diff
@@ -93,7 +93,7 @@ def initialize(inspec)
     end
   end
 
-  class LinuxFileSystemResource < FsManagement
+  class UnixFileSystemResource < FsManagement
     def info(partition)
       cmd = inspec.command("df #{partition} -PT")
```

Comment:
This sort-of doesn't work on FreeBSD 11:
```
[vagrant@bazinga ~]$ df / -PT
df: -PT: No such file or directory
Filesystem   1K-blocks   Used    Avail Capacity  Mounted on
/dev/ada0s1a  30450780 820904 27193816     3%    /
[vagrant@bazinga ~]$ echo $?
1
```

My Vagrantfile was simply:
```
Vagrant.configure("2") do |config|
  config.vm.box = "generic/freebsd11"
end
```

Reply:
Filed https://github.com/inspec/inspec/issues/4868